apps/builder/app/builder/features/pages/image-info.tsx

```diff
@@ -44,7 +44,11 @@ export const ImageInfo = ({ asset, onDelete }: ImageInfoProps) => {
         >
           <Grid flow={"column"} gap={1} align={"center"}>
             <ImageIcon />
-            <Text truncate variant={"labelsTitleCase"}>
+            <Text
+              truncate
+              variant={"labelsTitleCase"}
+              css={{ overflow: "hidden" }}
```

Comment:
oh shit, we changed truncate() utility to overflow: clip and this is why it broke ... looks like I broke it here by changing to clip and thinking clip works for truncation too .... https://github.com/webstudio-is/webstudio/commit/b68d29bbca74e043df874a59f78a8c4f2eac1f69

@TrySound @istarkov JFYI we need to be careful with clip, doesn't always work

@JayaKrishnaNamburu please change the original truncate utility back to hidden instead

Reply:
wow